agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4780,6 +4782,22 @@ bool slave_thread::handle_ap_capability_query(Socket *sd, ieee1905_1::CmduMessag
     return true;
 }
 
+bool slave_thread::handle_client_association_request(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx)
+{
+    //TODO - this is a stub handler for the purpose of controller certification testing,
+    //       will be implemented later on agent certification
+    const auto mid = cmdu_rx.getMessageId();
+    LOG(DEBUG) << "Received CLIENT_ASSOCIATION_CONTROL_REQUEST_MESSAGE, mid=" << std::dec
+               << int(mid);
+
```

Comment:
Same comment as in #271: ideally, you should parse out the Client Association Control Request TLV, match the BSSID, and only ack if it corresponds to this slave. Still: nitpick.